### ISSUE TITLE
GPL3 license violation by other app

### ISSUE THREAD

**Issue by vertigo220** [ligi/SurvivalManual#262]
"Survival Manual Offline" by bulltdapps appears to be a clone of this app with some various modifications. While GPLv3 allows for the use and modification of code under its license, it requires any code built from it to also be published under GPL3 and for changes to be indicated, neither of which appear to be done here. AFAICT, this "developer" has taken the codebase of this app, applied a few minor tweaks, and remarketed it with ads in order to profit off the work of Ligi and others, in which case it should be dealt with.

Note that many reviewers also criticized "Offline Survival Guide" by Priyo Islam, but that one is NOT plagiarized from this. People just don't seem to understand that both share the same source material.

**Comment by ligi** [comment]
Thanks - contacted the author of "Survival Manual Offline" - if they do not react I will DMCA

**Comment by ligi** [comment]
seems they took down the app - really weird that they rather take the app down than publishing the source code
Anyway - I think I can close this now. Thanks for the pointer.

**Comment by vertigo220** [comment]
Well this is strange. I had it installed still, and despite it supposedly being taken down, I saw it was renamed to "Manual de Sobrevivencia" and there was an update for it. So I did it, and now the entire app is in Spanish (which is especially odd considering the developer's address is Bulgaria). I couldn't find it by searching for it, though, and I shared the [link](https://play.google.com/store/apps/details?id=com.aplications.adimov.suviveres&authuser=0) and tried going to it in the browser and it doesn't find anything. So not really sure if it's down or not. It doesn't show up under the developer's apps, either, so I'm guessing it is. Very bizarre.
